### ISSUE TITLE
how to split the dataset from train and test and calculate accuracy?

### ISSUE THREAD

**Issue by anupamadeo** [NielsRogge/Transformers-Tutorials#62]
I am trying LayoutLMv2 on a new dataset for question answering using the same code given in the notebook. I wanted to train and test it but not able to create batches for test. I am also new to pytorch , I have worked with tensorflow. Kindly help
